File: eBarberShop/UI/ebarbershop_mobile/lib/models/uposlenik/uposlenik.dart
import 'package:json_annotation/json_annotation.dart';

part 'uposlenik.g.dart';

@JsonSerializable()
class Uposlenik {
  int uposlenikId;
  String ime;
  String prezime;
  String kontaktTelefon;
  String? email;
  String? adresa;
  double prosjecnaOcjena;

  Uposlenik(this.uposlenikId, this.ime, this.prezime, this.kontaktTelefon, this.email, this.adresa, this.prosjecnaOcjena);

  factory Uposlenik.fromJson(Map<String, dynamic> json) =>
      _$UposlenikFromJson(json);

  Map<String, dynamic> toJson() => _$UposlenikToJson(this);

}

File: eBarberShop/eBarberShop.Model/Uposlenik.cs
namespace eBarberShop.Model
{
    public class Uposlenik
    {
        public int UposlenikId { get; set; }
        public string Ime { get; set; }
        public string Prezime { get; set; }
        public string KontaktTelefon { get; set; }
        public string? Email { get; set; }
        public string? Adresa { get; set; }
        public double? ProsjecnaOcjena { get; set; }
    }
}
